### ISSUE TITLE
Move to the mongoid organization?

### ISSUE THREAD

**Issue by dblock** [Polarion/mongoid_relations_dirty_tracking#5]
Would you like to move this repo to the github/mongoid org? See http://mongoid.github.io/.

It could use some more active maintenance, support for Mongoid 6, etc.

(I'm coming from https://github.com/mongoid/mongoid-history/issues/187).

**Comment by davidsevcik** [comment]
Hello, sure, sounds like good idea.

**Comment by davidsevcik** [comment]
I cannot change the repo ownership as I don't work for Polarion anymore and cannot administer the repo. But as it's under MIT licence I don't see problem to clone the repo for mongoid.

**Comment by dblock** [comment]
Maybe @pupca or @srcx can help?

**Comment by dblock** [comment]
I've emailed ^ today, if we don't get a response lets make a fork in mongoid, I'll take care of it @davidsevcik.

**Comment by mateuspontes** [comment]
@dblock let's do it?

**Comment by dblock** [comment]
Yes please! Just transfer this repo to me and I'll move it to mongoid immediately.

**Comment by mateuspontes** [comment]
I'm not the owner, in this case I think we should fork/move @bopm https://github.com/bopm/mongoid_relations_dirty_tracking

**Comment by dblock** [comment]
The whole point was to not fork, lets wait to hear from these guys? We have an email thread.

**Comment by bopm** [comment]
I am not an owner of those changes, I just borrow them from @Megawolf492 who moved in a different direction with his fork after those. So I am ok with any outcome. Especially with some which will allow me to return stable release to Gemfile instead of link to my github :)

**Comment by pupca** [comment]
I am waiting for a confirmation from within my organization to OK this.
Give us a little more time please.

Martin

2017-04-25 0:46 GMT+02:00 Sergey Moiseev <notifications@github.com>:
